### PR TITLE
[WIP] Add Hot to contribute page [en, pt-BR]

### PR DESCRIPTION
**Pull request by ktquez**
Draft: https://vue-a11y-next.surge.sh/project/how-to-contribute.html

### CODE REVIEW

File: src/project/how-to-contribute.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+
+The project is supported by people who love Vue.js and accessibility, and who know the importance of the web being more inclusive.
+
+Your support is very important so that we can devote more time to creating content, keeping plugins and components up to date, in addition to funding content creators for their time dedicated to creating robust and well-tested posts.
+
+<p>
+We have separated some rewards for our Open Collective supporters.
+<br>
+<a href="https://opencollective.com/vue-a11y/donate" target="_blank" rel="noopener" style="background-color: initial; width: 300px;">
+  <img src="https://opencollective.com/vue-a11y/donate/button@2x.png?color=blue" style="box-shadow: none; display: inline;" />
```
Comment (marcus-herrmann) [inline src/project/how-to-contribute.md:59]: Needs alt

File: src/pt/projeto/como-contribuir.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+O projeto é apoiado por pessoas que amam Vue.js e acessibilidade, e que sabem da importância da web ser mais inclusiva.
+
+Seu suporte é muito importante para que possamos nos dedicar mais tempo à criação de conteúdo, manter plugins e componentes atualizados, além de financiar os criadores de conteúdo pelo seu tempo dedicado à criação de posts robustos e bem testados.
+
+<p>
+Separamos algumas recompensas para nossos apoiadores do Open Collective.
+<br>
+<a href="https://opencollective.com/vue-a11y/donate" target="_blank" rel="noopener" style="background-color: initial; width: 300px;">
+  <img src="https://opencollective.com/vue-a11y/donate/button@2x.png?color=blue" style="box-shadow: none; display: inline;" />
```
Comment (marcus-herrmann) [inline src/pt/projeto/como-contribuir.md:59]: See above

File: src/project/how-to-contribute.md (hunk 1 above)
Comment (ktquez) [reply src/project/how-to-contribute.md]: 🤦‍♂️ I copied/pasted and forgot to check.

Thanks @marcus-herrmann

**Comment by ktquez** [comment]
@marcus-herrmann 
Do you think we should add any more information to this page?
Or can we publish and add more information when necessary?

**Comment by marcus-herrmann** [comment]
@ktquez Let's start with this content and improve incrementially, for example linking to the Vue 3 Styleguide once it's available. But I think the most important contents (how to contribute, starting points, etc( are already there

File: src/project/how-to-contribute.md [hunk 3]
```diff
@@ -1,5 +1,68 @@
...
+
+Your support is very important so that we can devote more time to creating content, keeping plugins and components up to date, in addition to funding content creators for their time dedicated to creating robust and well-tested posts.
+
+<p>
+We have separated some rewards for our Open Collective supporters.
+<br>
+<a href="https://opencollective.com/vue-a11y/donate" target="_blank" rel="noopener" style="background-color: initial; width: 300px;">
+  <span class="sr-only">Donate to our collective (opens in a new tab)</span>
+  <img aria-hidden="true" src="https://opencollective.com/vue-a11y/donate/button@2x.png?color=blue" style="box-shadow: none; display: inline;" alt="" />
+</a>
```
Comment (marcus-herrmann) [inline src/project/how-to-contribute.md:61]: Now seems a little bit overengineered to me (aria-hiding and null alt-ing image, establishing sr-only text). Why not put `Donate to our collective (opens in a new tab)` in the images alt?

Comment (ktquez) [reply src/project/how-to-contribute.md:60]: Thanks @marcus-herrmann 
Done!

**Review by amycruz97 — COMMENTED** [review]
looks great

**Pull request merged**